web/src/styles/wiki-shell.css

```
@@ -0,0 +1,86 @@
+}
+
+.wiki-tab-badge {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 18px;
+  height: 18px;
+  padding: 0 6px;
+  background: #C78A1F; /* shared with notebook amber so reviews read as notebook-derived */
+  color: #ffffff;
+  font-family: 'Geist Mono', 'SFMono-Regular', Menlo, monospace;
```

Stylelint (17.7.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)